pkg/nsx/services/realizestate/realize_state.go

```diff
@@ -79,30 +79,25 @@ func (service *RealizeStateService) CheckRealizeState(backoff wait.Backoff, inte
 	})
 }
 
-func (service *RealizeStateService) GetPolicyTier1UplinkPortIP(intentPath string) (string, error) {
-	results, err := service.NSXClient.RealizedEntitiesClient.List(intentPath, nil)
+func (service *RealizeStateService) GetPolicyServiceInterfaceSNATIP(realizedPath string) (string, error) {
```

Comment:
one more comment: 
I noted that CTGW is using gateway-interface instead of service-interface.  To be generic,  I feel that we rename func to : GetPolicyInterfaceSNATIP might be better.

Reply:
ok, fixed